src/Helper/HeadLink.php

```diff
@@ -201,12 +197,14 @@ public function __call($method, $args)
     /**
      * Check if value is valid
      *
-     * @param  mixed $value
+     * @internal This method will become private in version 3.0
+     *
+     * @param mixed $value
      * @return bool
      */
     protected function isValid($value)
     {
-        if (! $value instanceof stdClass) {
+        if (! is_object($value)) {
```

Comment:
Hmm, this seems like a BC break

Reply:
For users (inheritors), behaviour hasn't changed - if anything, it's a little more relaxed now as you could more easily provide a value object, providing it has the expected public properties. All of these similar `isValid` methods do the same thing: Is it an object and does it have the properties required? No behaviour has changed.